sdv/multi_table/base.py

```diff
@@ -359,3 +360,48 @@ def get_learned_distributions(self, table_name):
         """
         synthesizer = self._table_synthesizers[table_name]
         return synthesizer.get_learned_distributions()
+
+    def add_constraints(self, constraints):
+        """Add constraints to the synthesizer.
+
+        Args:
+            constraints (list):
+                List of constraints described as dictionaries in the following format:
+                    * ``constraint_class``: Name of the constraint to apply.
+                    * ``table_name``: Name of the table where to apply the constraint.
+                    * ``constraint_parameters``: A dictionary with the constraint parameters.
+
+        Raises:
+            SynthesizerInputError:
+                Raises when the ``Unique`` constraint is passed.
+        """
+        if self._fitted:
+            warnings.warn(
```

Comment:
Adding the warning here so it only shows once. If I used the SingleTable's add_constraints warning, it would show for every constraint in the list.

I'll add a test for the warning after you thumbs up this.